File: backend/src/main/java/com/atl/bakuboutiquehub/dto/BoutiqueDTO.java
package com.atl.bakuboutiquehub.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class BoutiqueDTO {
    private Long id;
    private String name;
    private String description;
    private String address;
    private String contactNumber;
    private String workingHours;
    private java.util.List<String> categories;
    private java.util.List<String> brands;
    private Long ownerId;
}

File: app/src/main/java/com/atl/bakuboutiquehub/network/model/BoutiqueDTO.kt
package com.atl.bakuboutiquehub.network.model

data class BoutiqueDTO(
    val id: Long? = null,
    val name: String,
    val description: String? = null,
    val address: String? = null,
    val contactNumber: String? = null,
    val workingHours: String? = null,
    val categories: List<String>? = null,
    val brands: List<String>? = null,
    val ownerId: Long? = null
)
